pixtools/clusters/noise_analysis_SD_kmeans_Clustering.py

```diff
@@ -0,0 +1,148 @@
+# Unlike other noiseplot file (by channel) this file will cluster the SDs, allowing for a mean square analysis
```

Comment:
Please could you move this module into a new folder `pixtools/noise/` rather than `pixtools/clusters/`? 'clusters' here refers to clusters of spikes as determined by kilosort, not clustering analyses.